### ISSUE TITLE
Corrupted image when duplicating lora

### ISSUE THREAD

**Issue by Padusarbor** [opparco/stable-diffusion-webui-composable-lora#7]
When I duplicate lora, the image becomes corrupted, both with and without two-shot.
![image](https://user-images.githubusercontent.com/32173962/221775506-13d1ce43-5ea6-4f08-87bd-1e126e46acdf.png)
![image](https://user-images.githubusercontent.com/32173962/221775649-2866089b-f544-4314-b135-47a45be62732.png)
Without two-shot
![image](https://user-images.githubusercontent.com/32173962/221776068-7c181dbd-5fb4-4103-bd40-cf46631a720d.png)
With two-shot
![image](https://user-images.githubusercontent.com/32173962/221776284-2c234088-8cc6-49de-a487-ba32d02de63b.png)

**Comment by basedholychad** [comment]
im using colab and i have the same issue as this.
Images keep gettings bleeding/corrupted.
anyone know how to fix?

**Comment by QR-0W** [comment]
yes, i came into this question too. I decided to not use this extension temporarily.

**Comment by prescience-data** [comment]
An alternative approach I've been working with sans `Composable LoRA` here: 
https://github.com/opparco/stable-diffusion-webui-two-shot/issues/29#issuecomment-1475094100
